glfw/src/Window.zig

```diff
@@ -162,9 +163,6 @@ pub const Hint = enum(c_int) {
     /// Context client API minor version hint.
     context_version_minor = c.GLFW_CONTEXT_VERSION_MINOR,
 
-    /// Context client API revision number hint.
-    context_revision = c.GLFW_CONTEXT_REVISION,
```

Comment:
The reason I had this here was that it is listed in the source as being both a hint and attribute:

https://github.com/glfw/glfw/blob/fb0f2f92a38c1d6a776ffeb253329f8d1c65694c/include/GLFW/glfw3.h#L1031-L1036

However, indeed, I can confirm from looking at the GLFW source that comment in their header seems incorrect. I will send a PR upstream to fix the comment.

Reply:
Sent https://github.com/glfw/glfw/pull/1992